website/docs/requirements.md

```diff
@@ -0,0 +1,45 @@
+---
+id: requirements
+title: Requirements
+sidebar_label: Requirements
+---
+
+In order for to have the smoothest development and setup experience we need some things to be installed on your computer.
+Please install this packages before you start the setup.
+
+1. [Node.js](https://nodejs.org/en/)
+2. [Composer](https://getcomposer.org/)
+3. [WP-CLI](https://wp-cli.org/)
+4. [Git](https://git-scm.com/)
+
+Make sure that you have all this packages installed and
+ready to use on your system. To make sure that they work run this commands:
+- `node -v`
+- `composer -v`
+- `wp --info`
+- `git --version`
+
+
+If it doesn't return any errors you are good to go.
+
+## Quick start
+Let's finally create a new theme!
+
+Navigate to your WordPress theme folder and run the following command:
+
+```bash
+npx create-wp-project
+```
+
+Script will prompt you for theme name and local development url (used for BrowserSync) and install a new theme:
```

Comment:
🆙

Reply:
This part is wrong here I have removed it